### PR TITLE
docs: Replace Operator steps with Redpanda Helm chart in labs

### LINKED ISSUE #1043: Replace Operator with Redpanda Helm chart in Labs
### Describe the Issue

We aren't recommending Operator for new deployments at the moment, so we should update our doc about [Labs](https://docs.redpanda.com/docs/labs/install-preview/) to use the Helm chart.

We can tell users to do something like this:

```
helm upgrade --install redpanda redpanda/redpanda \
  --set image.repository=vectorized/redpanda-unstable \
  --set image.tag=v22.3.11-rc1
```

### Updates to existing documentation

https://docs.redpanda.com/docs/labs/install-preview/

### PR DESCRIPTION
**Pull request by JakeSCahill**
Review deadline: January 20

Fixes #1043

### CODE REVIEW

**Review by RafalKorepta — APPROVED** [review]
Only one NIT.

File: docs/labs/data-transform.mdx [hunk 1]
```diff
@@ -465,6 +465,10 @@ The [API package](https://www.npmjs.com/package/@vectorizedio/wasm-api) is publi
 
 ---
 
+## Share your feedback
```
Comment (Feediver1) [inline docs/labs/data-transform.mdx:468]: This is a bit of a conundrum given that every doc topic has a Share your feedback and doc PR link in the footer. What is the reasoning here?

File: docs/labs/install-preview.mdx [hunk 2]
```diff
@@ -1,165 +1,158 @@
...
+  <Tabs>
+  <TabItem value="install-fedora" label="Fedora/RedHat/Amazon Linux" default>
 
-Current lab features are:
+  ```bash
+  ## Run the setup script to download and install the unstable repo
+  curl -1sLf 'https://dl.redpanda.com/E4xN1tVe3Xy60GTx/redpanda-unstable/setup.rpm.sh' | 
+  sudo -E bash && \
+  ## Use yum to install redpanda
+  sudo yum install redpanda
```
Comment (JakeSCahill) [inline docs/labs/install-preview.mdx:33]:
```suggestion
  sudo yum install redpanda -y
```

**Comment by Feediver1** [comment]
I'm curious to know--"Verify the transform" uses numbered steps, but none of the other tasks do. Is it the case that the user is not required to complete the steps in order? I find it easier to follow tasks when in numbered steps, but that could just be my preference. Knowing the step number in a task could be helpful when a customer is talking to support.

**Review by Feediver1 — APPROVED** [review]
Left a comment about how tasks are presented (paras vs steps). Your call.

**Comment by JakeSCahill** [comment]
> I'm curious to know--"Verify the transform" uses numbered steps, but none of the other tasks do. Is it the case that the user is not required to complete the steps in order? I find it easier to follow tasks when in numbered steps, but that could just be my preference. Knowing the step number in a task could be helpful when a customer is talking to support.

I agree. I think there's a larger amount of work to do with the Data Transforms doc, but I didn't want to overload this PR since it's supposed to only fix #1043. For example, I created these issues: #1051  #1052

**Pull request merged**